tests/test_workflows.py

```diff
@@ -489,44 +489,27 @@ def test_extract_text(url, name, strict, exception):
             "https://corpora.tika.apache.org/base/docs/govdocs1/957/957304.pdf",
             "tika-957304.pdf",
         ),
-    ],
-)
-def test_compress_raised(url, name):
-    data = BytesIO(get_pdf_from_url(url, name=name))
-    reader = PdfReader(data)
-    # no more error since iss #1090 fix
-    for page in reader.pages:
-        page.compress_content_streams()
-
-
-@pytest.mark.enable_socket()
-@pytest.mark.slow()
```

Comment:
Note for myself: The slowest test now is the one with https://corpora.tika.apache.org/base/docs/govdocs1/950/950337.pdf-tika-950337.pdf with about 1.4s. We only want to mark tests with > 5s with `slow`, so removing this is fine.